### PR TITLE
Remove word-break: break-all from code element

### LINKED ISSUE #3526: The code element breaks words
## Description
When the `code` element is too long to fit on a line it breaks within the word, making the command hard to read/understand.

## Reproduce
This has been patched on snapcraft.io but see this issue for more context: https://github.com/canonical-web-and-design/snapcraft.io/issues/3377

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Removed `word-break: break-all;` from `code` elements.

Fixes #3526 

## QA

- Open [demo](https://vanilla-framework-3535.demos.haus/docs/examples/layouts/documentation)
- Scroll to the "Notifications" heading
- Reduce the width of your browser, see that as you do, the code element within wraps, but wraps whole words rather than breaking words

### CODE REVIEW

**Review by anthonydillon — APPROVED** [review]
LGTM

**Review by bartaz — APPROVED** [review]
LGTM

File: scss/_base_code.scss [hunk 1]
```diff
@@ -25,10 +25,7 @@ $code-sidebar-width: calc(#{$sph-inner} + (4 * #{$digit-width})) !default;
   samp {
     background-color: $color-mid-x-light;
     box-decoration-break: clone;
-    box-shadow: 0 0 0 0.25rem $color-mid-x-light;
-    margin-left: 0.25rem;
-    margin-right: 0.25rem;
-    word-break: break-all;
+    padding: 0.25rem;
```
Comment (bartaz) [inline scss/_base_code.scss:28]: @lyubomir-popov Do we have a var for such padding on inline element?

Comment (lyubomir-popov) [reply scss/_base_code.scss:28]: $spv-inner--x-small

Comment (lyubomir-popov) [reply scss/_base_code.scss:28]: we should add $sph-inner--x-small as it is horizontal in this case

Comment (sowasred2012) [reply scss/_base_code.scss]: Thanks, just added

**Pull request merged**